os-eap-launch/added/launch/datasource-common.sh

```diff
@@ -494,8 +494,13 @@ function inject_datasource() {
 
   url=$(find_env "${prefix}_URL")
 
-  case "$db" in
+  IFS=. read -r -a java_version <<< "${JAVA_VERSION}"
+  java_maj_version=${java_version[0]:-8}
+  # JDK >= 11 doesn't have the default drivers present.
+  if [ ${java_maj_version} -lt 11 ]; then
+    case "$db" in
     "MYSQL")
+      log_warning "Datasource ${service_name} is using the bundled MySQL driver. This driver has been deprecated and will be dropped in a future release. Please see the product documentation for more information on installing database drivers."
```

Comment:
https://issues.jboss.org/browse/JBEAP-16626 is triaged for 7.2.2, not 7.2.1, so let's not do this yet.

Also, a commit for it should reference CLOUD-3143.

Reply:
OK, this change is dropped. I'll reply to your other comment below.